config.md

```diff
@@ -0,0 +1,224 @@
+---
+title: Configuration
+layout: sidebar
+category: intro
+---
+
+## Configuration
+
+Testium uses [`rc`](https://www.npmjs.com/package/rc) for loading configuration.
+Be aware that command line arguments won't work
+since most test runners (e.g. `mocha`) have their own argv handling.
+
+### Top-Level Options
+
+#### `browser`
+
+The name of the browser to use.
+Expects a valid webdriver `browserName`.
```

Comment:
Might be cool to say or link to info on how to get a list of browserNames supported by your installation.